src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -5349,4 +5349,26 @@ public static boolean isPasswordValidatorReconfigurationEnabled()
     {
         return conf.password_validator_reconfiguration_enabled;
     }
+
+    public static long getInvalidClientDisconnectPeriod(TimeUnit unit)
+    {
+        return conf.invalid_client_disconnect_period.to(unit);
+    }
+
+    public static void setInvalidClientDisconnectPeriod(TimeUnit unit, long duration)
+    {
+        logger.info("invalid_client_disconnect_period set to {} {}", duration, unit);
```

Comment:
I find it useful to know what was the previous value as well during debugging, so if we could include that, it'd be great. i.e. `invalid_client_disconnect_period set from <previous> to <new>`